.ci/pipeline/build-and-test-lnx.yml

```diff
@@ -24,7 +24,7 @@ steps:
     displayName: 'System info'
   - script: |
       conda update -y -q conda
-      if [ $(echo $(PYTHON_VERSION) | grep '3.7') ] || [ $(echo $(PYTHON_VERSION) | grep '3.11') ]; then export DPCPP_PACKAGE="dpcpp-cpp-rt>=2023.1.0"; else export DPCPP_PACKAGE="dpctl>=0.14 dpcpp-cpp-rt>=2023.1.0"; fi
+      if [ $(echo $(PYTHON_VERSION) | grep '3.7') ] || [ $(echo $(PYTHON_VERSION) | grep '3.11') ]; then export DPCPP_PACKAGE="dpcpp-cpp-rt>=2023.2.0"; else export DPCPP_PACKAGE="dpctl>=0.14.5 dpnp>=0.12.1 dpcpp-cpp-rt>=2023.2.0"; fi

```

Comment:
We don't need dpnp on building, so there is no reason install it on conda creation step, I think

Reply:
dpnp installation in testing stage will mix conda and pip commands which can result in env consistency problems in future. I prefer to preserve next order: conda env creation -> conda packages installation -> pypi packages installation